jcli/src/jcli_app/block/DOCUMENTED_EXAMPLE.yaml

```diff
@@ -54,21 +54,22 @@ blockchain_configuration:
   # The speed to update the KES Key in seconds
   kes_update_speed: 43200 # 12hours
 
-# The initial deposits present in the blockchain.
-#
-# It can be UTxO addresses or account (if allow_account_creation is set).
-initial_funds:
-  # this is the personal address of the developers, have a good heart and
-  # leave a good initial deposit for them
-  - address: {initial_funds_address}
-    value: 10000
+# The initial state of the ledger. Each item is applied in order of this list
+initial:
+  # The initial deposits present in the blockchain.
+  #
+  # It can be UTxO addresses or account (if allow_account_creation is set).
```

Comment:
Can we remove `allow_account_creation` mentioning, or it is still a valid directive?

Reply:
No, it's not, they exist only in docs, which seem outdated